crates/ironrdp-session/src/active_stage.rs

```diff
@@ -245,6 +249,10 @@ impl ActiveStage {
 
         None
     }
+
+    pub fn encode_dvc_messages(&mut self, message: Vec<SvcMessage>) -> SessionResult<Vec<u8>> {
+        self.process_svc_processor_messages(SvcProcessorMessages::<DrdynvcClient>::new(message))
```

Comment:
naming: argument `messages` (plural)

note: This function seems to be for encoding **DVC** messages, but it takes `SvcMessage` values. Maybe the abstraction for SVC messages was reused as is for DVC messages, but this can be confusing.